.github/workflows/check_csv_data_integrity.yaml

```
@@ -0,0 +1,117 @@
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+
+      - name: Check CSV length preservation
+        run: |
+          echo "🔍 Checking CSV length preservation for edited files..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)